_data/toc.yml

```diff
@@ -154,6 +154,7 @@
       - Authentication
       - Commands
       - Events
+    - Layout Embedded
```

Comment:
Should this be changed to `Embedded Layout` as of [page's title](https://github.com/RocketChat/docs/pull/681/files#diff-21a9468c9abb601c7f3650003dd3ae19R1)